### PR TITLE
added comment for unsigned int functions

### PR DESCRIPTION
**Pull request by spatsch**

### CODE REVIEW

File: tigon-sql/cfg/external_fcns.def [hunk 1]
```diff
@@ -38,7 +45,11 @@
 //          Function for casting
 ///////////////////////////////////////////////////////////
 
- 	int FUN [LFTA_LEGAL,COST FREE] non_temporal(int);
+
+// REMARK: UNSIGNED FUNCTIONS RETAINED FOR GSQL STAND
+// ALONE USE CURRENTLY NOT SUPPORTED BY TIGON STREAMS
+
+    int FUN [LFTA_LEGAL,COST FREE] non_temporal(int);
```
Comment (gokulavasan) [inline tigon-sql/cfg/external_fcns.def:52]: Does this function need this Remark? It seems to return a signed variable?

**Comment by gokulavasan** [comment]
@spatsch Oliver, can you address my comment. We can merge this right after that.

**Comment by spatsch** [comment]
I moved that function in front of the remark. Also copied the remark in a couple of more places. Not sure how exactly you want to structure it. Copying it in front of every function with an unsigned will clutter up the file quite a bit so I copied it in front of any section with unsigned functions in it. Let me know if that looks good or if you want to change it.

**Comment by gokulavasan** [comment]
Oh I see. That makes sense. LGTM :+1:

**Pull request merged**